lib/src/navbar/navbar.component.ts

```diff
@@ -18,7 +19,38 @@ export class NavbarComponent implements OnInit {
     constructor() {
     }
 
+    menuOpen: boolean = false;
+    hamburger: string = 'inline';
+    hamburgerClose: string = 'none';
+
+    @ContentChildren(NavbarMenuComponent) mobileMenu: QueryList <NavbarMenuComponent>;
+
     ngOnInit() {
     }
 
+    toggleMobileMenu() {
+        if( this.mobileMenu.first ) {
+            if( this.menuOpen ) {
+                this.mobileMenu.first.hide();
+                this.menuOpen = false;
+                this.hamburger = 'inline';
+                this.hamburgerClose = 'none';
+            }
+            else {
+                this.mobileMenu.first.show();
+                this.menuOpen = true;
+                this.hamburger = 'none';
+                this.hamburgerClose = 'inline';
+            }
+        }
+    }
+
+    menuClick( event ) {
+        let clickTarget: string = event.target.classList[0];
```

Comment:
I suppose we know there will only be one class on the target, but this might be fragile if a new class is ever added. We may want to consider looping over the classes to find the right one?

Reply:
I think we're probably safe on this one because there really shouldn't even be more than one mobile menu.  Unless I'm not thinking of a situation where it might be useful to have more than one declared?